examples/data-api-builder/CommunityToolkit.Aspire.Hosting.Azure.DataApiBuilder.AppHost/dab-config.json

```diff
@@ -23,6 +23,9 @@
         ],
         "allow-credentials": false
       },
+      "authentication": {
+        "provider": "StaticWebApps"
```

Comment:
I think the provider should be the simulator, not SWA, for local dev.

Reply:
Yes, provider should be `Simulator` for local development. When no authN section is provided, DAB defaulted to SWA assuming DAB was being deployed in SWA.

Issue in #363 should be opened on DAB's repo as that does look to be a problem where excluding an auth section from dab config triggers in the referenced error.

Reply:
Created https://github.com/Azure/data-api-builder/issues/2513 to track it in the DAB repo @seantleonard

Reply:
Also seems odd that dab init defaults to SWAAuth